app/views/works/form.html.erb

```diff
@@ -1,5 +1,6 @@
 <h1 class="h2 mb-3"><%= @work_form.title || 'Untitled deposit' %></h1>
- <div class="d-flex tabbable-panes">
+ <%= render Elements::AlertComponent.new(variant: :danger, value: I18n.t('works.edit.errors.validation')) if @work_form.errors.present? %>
```

Comment:
I wonder if it would be possible to use a `render?` in the component as we have elsewhere so we don't use the inline conditional here. Not a blocker, and could easily be a later refactor if we decide that pattern would work here later.

Reply:
The alert component is generic; it doesn't know anything about work forms or errors. (I had the same thought too.)